### PR TITLE
[Documentation][Cookbook] How to customize Sylius Checkout?

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| BC breaks? | no |
| Related tickets | partially #5275 |
| License | MIT |

### CODE REVIEW

File: docs/cookbook/checkout.rst [hunk 1]
```diff
@@ -0,0 +1,292 @@
+How to customize Sylius Checkout?
+=================================
+
+Why would you override the Checkout process?
+--------------------------------------------
+
+This is a common problem for many Sylius users. Sometimes the checkout process we have designed is not suitable for your custom business needs.
+therefore you need to learn how to modify it, when you will need to for example:
```
Comment (Zales0123) [inline docs/cookbook/checkout.rst]: ``Therefore``

**Comment by Zales0123** [comment]
Pretty nice for me 👍  But as we've said on today's stand up - we should think about describing some more sophisticated example, as single page checkout, which is quite common use case.

File: docs/cookbook/checkout.rst [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                <i class="arrow right icon"></i>
+                {{ 'sylius.ui.next'|trans }}
+            </button>
+        </div>
+    </div>
+
+Overwrite routing for Checkout
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unfortunately there is no better way - you have to overwrite the whole routing for Checkout.
```
Comment (TheMadeleine) [inline docs/cookbook/checkout.rst]: @pjedrzejewski, @michalmarcinkowski wdyt?

Comment (michalmarcinkowski) [inline docs/cookbook/checkout.rst]: After import we would also have `sylius_shop_checkout_select_shipping` route available and that's wrong. I would say that if we customize checkout it's better to copy the whole routing and have full control over it. Same as we do with the state machine.

Comment (michalmarcinkowski) [inline docs/cookbook/checkout.rst]: It's even a Symfony best practice http://symfony.com/doc/current/bundles/override.html#routing

**Comment by pjedrzejewski** [comment]
Thank you Magda! :)

**Pull request merged**